dependency-manifest.yml

```diff
@@ -33,6 +33,10 @@ dependencies:
       version: 1.16.0
       source: https://github.com/cyrilgdn/terraform-provider-postgresql/archive/v1.16.0.zip
       source_sha256: 4e006eb9c8a4a0287dbbd58a54aa3adaad1bcd098700126b9f48a0aa31bafb6d
+    - name: terraform-provider-csbpg
```

Comment:
This is an automatically generated file so there's no need to update it in future.